department-of-veterans-affairs/va.gov-cms
# VACMS-5488: Enable menu force on multiple content types.

## Description
See #5488

## Testing done
Behat

## QA steps
 - Visually verify content types defined in task are set to menu_enable 1 in config.

### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
